src/app.js

```diff
@@ -32,6 +33,25 @@ function parsePrototypeFromReferrer(referrer) {
   return "";
 }
 
+function bodyContentBasedOnDynos(dynosCount) {
+  if (dynosCount == -1) {
+    return `
+    <p class="govuk-body">This prototype is not currently deployed to Heroku.</p>
+    <p class="govuk-body">It may have been undeployed to conserve resources.</p>
+    `
+  } else if(dynosCount == 0) {
+    return `
+    <p class="govuk-body">This prototype is deployed to Heroku but is turned off.</p>
+    <p class="govuk-body">It may have been turned off to conserve resources.</p>
+    `
+  } else {
+    return `
+    <p class="govuk-body">It may have been turned off to conserve resources.</p>
+    <p class="govuk-body">Or, this could be due to an error within the prototype preventing it from starting.</p>
```

Comment:
I reckon if we know there are some dynos associated then we shouldn't say it might have been turned off 

Maybe something like "This prototype is turned on but it failed to start up due to an error, please check the last change you deployed" but that content bits might be a holly / annabelle sign-off thing